debian/rules

```diff
@@ -12,9 +12,6 @@
 %:
 	dh $@ 
 
-override_dh_missing:
-	dh_missing --fail-missing --exclude=appcenter.blacklist
```

Comment:
We still want `dh_missing --fail-missing`

Reply:
I was hoping this might fix the build errors from Houston :p It's complaining about a missing file even though it doesn't exist :grimacing: